catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraDescription.scala

```diff
@@ -0,0 +1,29 @@
+package uk.ac.wellcome.transformer.transformers.sierra
+
+import uk.ac.wellcome.models.Agent
+import uk.ac.wellcome.transformer.source.{MarcSubfield, SierraBibData}
+
+trait SierraDescription {
+
+  def getDescription(bibData: SierraBibData): Option[String] = {
```

Comment:
The big change I’ll want in the final version is a comment, similar to that on getPublisher and getTitle, that explains the _intent_ of the transformation below. A tidied-up version of Silver’s notes would be fine, noting anywhere you’ve made assumptions or deviations from the spec.